website/versioned_docs/version-2.9.0/getting-started/installation-linux.md

```diff
@@ -13,7 +13,14 @@ Before you are ready to run Saleor, you will need additional software installed
 
 ### Python 3
 
-Saleor requires Python version 3.6 or later. A compatible version comes pre-installed with most current Linux systems. If not, consult your Linux distributor for instructions on how to install Python 3.6 or 3.7.
+Saleor requires Python version 3.7 or later. A compatible version comes pre-installed with most current Linux systems. If not, consult your Linux distributor for instructions on how to install Python 3.7 or 3.8.
```

Comment:
Not very true... For example, most Ubuntu comes with Python 3.5 as default Python 3

Reply:
I ran into the issue on ubuntu 18.04 that has python 3.6.9 and received an import error about dataclasses when trying to migrate the database. What I found was dataclasses was introduced in python 3.7, and when trying with that fix everything worked. 

Is there a way I can get it working with earlier versions of python with workarounds? I'm very interested in learning that type of stuff but still new. I will change it back to the way it was though my apologies

Reply:
Oh I see your point now

Reply:
> Is there a way I can get it working with earlier versions of python with workarounds? I'm very interested in learning that type of stuff but still new. I will change it back to the way it was though my apologies

No, that wouldn't be possible. Support for 3.6 and below were dropped voluntary. But Ubuntu uses the SRU update system rather than rolling, requiring to manually install the specific version of Python 3 (rolling).

Reply:
Gotcha. So in the documentation do you want to include more specific details about installing a working version of python on different distributions?

Reply:
I don't think it's needed, users most likely know how to install a package on Linux or how to Google it. We can leave to simply telling it requires python 3.7 or higher.